Makefile

```diff
@@ -102,6 +102,8 @@ build-push-ee-query-service:
 	--build-arg LD_FLAGS="$(LD_FLAGS)" --tag $(REPONAME)/$(QUERY_SERVICE_DOCKER_IMAGE):$(DOCKER_TAG) .
 
 dev-setup:
+	sudo apt update
+	sudo apt install sqlite3
```

Comment:
This doesn't work on macOS, or anywhere the `apt` package manager doesn't exist

Reply:
it's for Linux, I'll just add a check to run these commands on Linux only.

Reply:
I was pointing out that you shouldn't assume the user operating system. Even with the modified check, I don't think it runs on any distro that is not Debian or its derivative. I would rather just add a note to make users install it themselves using the appropriate package manager or take the distro into account.

Reply:
This will run on any Linux Distros.

Reply:
@axaysagathiya This only works on Debian system or Linux which uses apt package manager. So maybe we should rather also add a note for the user to install sqlite themselves

Reply:
Added SQLite3 installation steps in CONTRIBUTING.md